Userland/Libraries/LibJS/Runtime/NumberPrototype.h

```diff
@@ -6,6 +6,8 @@
 
 #pragma once
 
+#include <LibCrypto/BigInt/SignedBigInteger.h>
+#include <LibCrypto/NumberTheory/ModularFunctions.h>
```

Comment:
Please add these includes to the cpp file instead if the header doesn't use them